src/Plugin/os2web/DataLookup/ServiceplatformenCPRExtended.php

```diff
@@ -111,7 +111,23 @@ public function lookup($cpr, $allowCprTestModeReplace = TRUE) {
         $cprResult->setPostalCode($address->aktuelAdresse->postnummer ?? '');
         $cprResult->setCity($address->aktuelAdresse->postdistrikt ?? '');
         $cprResult->setMunicipalityCode($address->aktuelAdresse->kommunekode ?? '');
-        $cprResult->setAddress($address->aktuelAdresse->standardadresse ?? '');
+
+        // Composing full address in one line.
+        $address = $cprResult->getStreet();
+        if ($cprResult->getHouseNr()) {
+          $address .= ' ' . $cprResult->getHouseNr();
+        }
+        if ($cprResult->getFloor()) {
+          $address .= ' ' . $cprResult->getFloor();
+        }
+        if ($cprResult->getApartmentNr()) {
+          $address .= ' ' . $cprResult->getApartmentNr();
+        }
+        if ($cprResult->getPostalCode() && $cprResult->getCity()) {
+          $address .= ', ' . $cprResult->getPostalCode() . ' ' . $cprResult->getCity();
+        }
+
+        $cprResult->setAddress($address ?? '');
```

Comment:
The formatting seems slightly off.  I believe it should be 

`street` `houseNr`, `floor`. `apartmentNr`, `postalCode` `city`

See A.1 and A.4 https://danmarksadresser.dk/om-adresser/saadan-gengives-en-adresse.

Reply:
we were pointed to the old version of CPR-looked created by ITK, see here:
https://github.com/itk-dev/os2forms_cpr_lookup/blob/develop/src/CPR/CprServiceResult.php#L154

this implementation is returning the address in the same way as it did for ITK implementation.

Please let me know if we want it differently